foundation-extensions/experience-platform-extension/README.md

```diff
@@ -165,7 +164,7 @@ Install other extensions based on your use-case:
 
 1. If your application requires user consent preferences collection and enforcement, install and configure the [Consent for Edge Network](../consent-for-edge-network) extension.
 2. Lifecycle extension now supports application lifecycle metrics collection for Edge Network. If you would like to start collecting this type of data, follow the installation instruction for [Lifecycle for Edge Network](../lifecycle-for-edge-network).
-3. If your application uses push notifications, see also the [Adobe Journey Optimizer](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-journey-optimizer) extension.
+3. If your application uses push notifications, see also the [Adobe Journey Optimizer](../../using-mobile-extensions/adobe-journey-optimizer) extension.
```

Comment:
link should be `../../foundation-extensions/adobe-journey-optimizer`

Reply:
I can only find journey-optimizer under using-mobile-extensions.